src-tauri/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "app"
+version = "0.1.0"
+description = "A Tauri App"
+authors = ["you"]
+license = ""
```

Comment:
GPL v3

Reply:
Ok. In Cargo.toml the format is [described here](https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields), in this case `GPL-3.0`.